src/js/bootstrap-slider.js

```diff
@@ -1148,9 +1148,15 @@ const windowIsDefined = (typeof window === "object");
 
 				this.handle1.style[this.stylePos] = positionPercentages[0]+'%';
 				this.handle1.setAttribute('aria-valuenow', this._state.value[0]);
+				if (isNaN(this.options.formatter(this._state.value[0])) ) {
+					this.handle1.setAttribute('aria-valuetext', this.options.formatter(this._state.value[0]));
+				}
 
 				this.handle2.style[this.stylePos] = positionPercentages[1]+'%';
 				this.handle2.setAttribute('aria-valuenow', this._state.value[1]);
+				if (isNaN(this.options.formatter(this._state.value[1])) ) {
+					this.handle2.setAttribute('aria-valuetext', this.options.formatter(this._state.value[1]));
```

Comment:
i wonder why this test is still failing (yet passes when the `describe` is not there)